modules/local/genome_uploader/webin_cli_upload/main.nf

```diff
@@ -0,0 +1,40 @@
+process WEBIN_CLI_UPLOAD {
+
+    label 'process_low'
+    tag "${id}"
+    stageInMode 'copy'
+    container "quay.io/biocontainers/ena-webin-cli:8.2.0--hdfd78af_0"
+
+    input:
+    tuple val(id), path(mag), path(manifest)
+
+    output:
+    tuple val(id), path("webin-cli.report") , emit: webin_report
+    tuple val(id), env(success_status)      , emit: upload_status
```

Comment:
3 comments on this file:
- Env variables should be quoted (https://www.nextflow.io/docs/latest/strict-syntax.html#process-env-inputs-and-outputs)
- It's a convention to use upper case for env variables (just saying I don't particually mind)
- Nextflow secrets?